source/backends/debian/debpkg.d

```diff
@@ -121,6 +121,20 @@ public:
         return pa;
     }
 
+    protected void extractPackage (const string dest = buildPath (tmpDir, name))
+    body
```

Comment:
Won't hurt :) But this function will throw an error anyway when the destination directory failed to be created, so I think just removing the `body {}` and go without contract would also be fine.